src/Components/SourcesSelect/index.js

```diff
@@ -65,20 +65,27 @@ const SourcesSelect = ({ setValidation, image }) => {
   }
 
   return (
-    <Select
-      ouiaId="select_account"
-      isOpen={isOpen}
-      onToggle={(openState) => setIsOpen(openState)}
-      selections={selected}
-      maxHeight="180px"
-      // TODO decide if to disable the select
-      // isDisabled={sources?.length === 1}
-      onSelect={onSelect}
-      placeholderText="Select account"
-      aria-label="Select account"
-    >
-      {sources && selectItemsMapper()}
-    </Select>
+    <>
+      <Select
+        ouiaId="select_account"
```

Comment:
can you add `validated=warning` prop if the selected source is invalid?

Reply:
I don't see any `validate` prop here: https://www.patternfly.org/components/menus/select/#props